docs/technical-docs/actions.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Python CI Workflow
+parent: Technical Documentation
+---
+
+# Python CI Workflow
+
+## Overview
+
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions) is a tool available to
```

Comment:
Please define:
Continuous integration
Linter/linting

Reply:
Actually, you link to Wikipedia for the definition of linting. so dont need to define that here. I like the way you link to Wikipedia for definitions throughout. Please keep doing that!

Please add a standalone definition of CI at the top of this doc though, as that is the focus (also include wikipedia link)